agentlib/modules/simulator.py

```diff
@@ -326,7 +326,12 @@ def model(self, model: Model):
                 f"but should be an instance of Model or a valid subclass"
             )
         self._model = model
-        self.model.initialize(t_start=self.config.t_start, t_stop=self.config.t_stop)
+        if self.config.t_start and self.env.offset:
+            self.logger.warning("config.t_start and env.offset are both non-zero. "
+                                "This may cause unexpected behavior. Ensure that this "
+                                "is intended and you know what you are doing.")
+        self.model.initialize(t_start=self.config.t_start + self.env.config.offset,
+                              t_stop=self.config.t_stop)
```

Comment:
Wasn't this a fix in #19 ?

Reply:
No, this is a change in #13 , but it is not merged yet. I would include it here and then close #14